tests/rails/llm/test_library_flow_files.py

```
@@ -0,0 +1,147 @@
+    package_dir = _package_dir(manifest)
+    parsed = {}
+    for version, file_names in (("1.0", manifest.spec.flows.v1_files), ("2.x", manifest.spec.flows.files)):
+        parsed[version] = []
+        for file_name in file_names:
+            path = package_dir / file_name
+            if not path.exists():
+                violations.append(f"{manifest.name}: missing {path}")
+                continue
+            try:
+                result = parse_colang_file(file_name, content=path.read_text(encoding="utf-8"), version=version)
+            except Exception as error:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)